HHS/Head-Start-TTADP
# [PROD] Turn on/off feature flags for all users in the admin UI plus a bug fix

## Description of change
Adds an ability to turn on/off flags for all users in the admin UI.
Fix an issue with saving a group.


## How to test
- navigate to the admin -> feature flags page
- made sure the buttons turn the flags on or off for all users as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1576
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1581



## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
